### PR TITLE
@craigspaeth Edits to the open source force post to hopefully improve clarity

### PR DESCRIPTION
**Pull request by zamiang**
Made a few edits to hopefully improve the flow of the post. Tried to start out direct and then have little transitions the lead into the next paragraph. Also reduced the # of prepositions yay -http://www.hemingwayapp.com/

### CODE REVIEW

File: source/_posts/2014-09-05-we-open-sourced-our-isomorphic-javascript-website.markdown [hunk 1]
```diff
@@ -39,10 +39,12 @@ We're so convinced this encapsulation is important that we've updated Ezel to [u
 
 ![Ocotcat](/images/2014-09-05-we-open-sourced-our-isomorphic-javascript-website/octocat.jpg)
 
-Even though Force isn't a library that can be leveraged by the community at large, we're hoping it serves as a nice reference for how we structure some of our apps. Unless there's something sensitive like licensed fonts, or API keys/secrets checked into the repo there's no reason we need to privatize our code. We'll even open source Artsy app-specific modules like [these backbone mixins](https://github.com/artsy/artsy-backbone-mixins) [this Artsy API authentication library](https://github.com/artsy/artsy-passport), or [this module](https://github.com/artsy/backbone-cache-sync) we use to cache server-side Backbone requests.
+Even though Force isn't a library, we have opensoured many of its componets and libraries. Before open sourcing Force, we open sourced app-specific modules such as [these backbone mixins](https://github.com/artsy/artsy-backbone-mixins) [this Artsy API authentication library](https://github.com/artsy/artsy-passport), or [this module](https://github.com/artsy/backbone-cache-sync) we use to cache server-side Backbone requests.
```
Comment (craigspaeth) [inline source/_posts/2014-09-05-we-open-sourced-our-isomorphic-javascript-website.markdown:42]: Typos: opensoured -> opensourced,  componets -> components. Inconsistent use of "opensourced" vs. "open sourced"

File: source/_posts/2014-09-05-we-open-sourced-our-isomorphic-javascript-website.markdown [hunk 2]
```diff
@@ -39,10 +39,12 @@ We're so convinced this encapsulation is important that we've updated Ezel to [u
...
+Opensourcing Force was pretty straighforward but we needed to make our sensitive keys/secrets private while not complicating development. To do this we wrote a .env file and uploaded it as a private gist that gets downloaded when setting up the app. We wanted to spread this open-source-by-default culture so we decided to update Ezel's configuration to be able to use a .env file in this way as well. This makes it easy keep your sensitive configuration data private while allowing the rest of your app code to be open source. You can read more about this in Ezel's [Build Scripts & Configuration docs](https://github.com/artsy/ezel#build-scripts--configuration).
```
Comment (craigspaeth) [inline source/_posts/2014-09-05-we-open-sourced-our-isomorphic-javascript-website.markdown:44]: "This makes it easy keep your sensitive..." -> "This makes it easy **to** keep your"

**Comment by craigspaeth** [comment]
Nice, great additions! That hemingwayapp is pretty sweet! I'll merge and fix up the things I pointed out along with @dblock's suggestions.

File: source/_posts/2014-09-05-we-open-sourced-our-isomorphic-javascript-website.markdown (hunk 2 above)
Comment (zamiang) [inline source/_posts/2014-09-05-we-open-sourced-our-isomorphic-javascript-website.markdown:44]: I check these for typos every time and yet…

**Comment by zamiang** [comment]
Thanks! Apologies about the typos. Maybe need to add grammar check or something to emacs :)

**Pull request merged**